config/defaults.json

```diff
@@ -618,5 +618,25 @@
     "version":        "",
     "parent_key":     "",
     "parent_value":   ""
+  },
+  {
+    "key": "success_center_phone",
```

Comment:
Small nitpick: can we make `success_center_phone` `success_phone` or make `success_email_to` `success_center_email_to` for naming consistency?